RootOutputerConfig.h

```diff
@@ -13,6 +13,7 @@ namespace cce::tf {
     int basketSize_=16384;
     int treeMaxVirtualSize_=-1;
     int autoFlush_=-1;
+    int cacheSize_=102400;
```

Comment:
I believe this type is used by all the ROOT*Outputer as well. Please update the others to use this.

Reply:
I didn't see any use of RootOutputConfig in the others. Do they get it through inheritance?